Configurations/unix-Makefile.tmpl

```diff
@@ -554,6 +554,10 @@ build_libs_nodep: $(LIBS) {- join(" ",map { platform->sharedlib_simple($_) // pl
 build_modules_nodep: $(MODULES)
 build_programs_nodep: $(PROGRAMS) $(SCRIPTS)
 
+# Convenience targets to skip the build of non-installable programs
+build_installable_sw: build_libs_nodep build_modules_nodep build_installable_programs link-utils
+build_installable_programs: $(INSTALL_PROGRAMS) $(SCRIPTS)
```

Comment:
This works for pristine builds, but this could be useful for developers too, so it might be better to use the same `dependmagic` stuff for these as for `build_sw` up by line 531

Reply:
Thanks for the quick review. Updated and shortened the target names to better fit into `make help`.